Makefile.local.common

```diff
@@ -67,6 +73,7 @@ COMPATIBILITY_FILES_PATTERN := \
 	src/Rewriter/Util/plugins/strategy_tactic.mli \
 	src/Rewriter/Util/plugins/strategy_tactic_plugin.@ML4_OR_MLG@ \
 	src/Rewriter/Util/plugins/strategy_tactic_plugin.mllib \
-        src/Rewriter/Util/plugins/RewriterBuildRegistry.v
+	src/Rewriter/Util/plugins/RewriterBuildRegistry.v \
+	src/Rewriter/Util/plugins/RewriterBuild.v
```

Comment:
src/Rewriter/Util/plugins/StrategyTactic.v also needs to be handled here, and split into versions